app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/CommandLine.scala

```diff
@@ -1,17 +1,22 @@
-package com.azavea.rf.ingest
+package com.azavea.rf.batch.ingest
 
 import scala.util._
-import java.util.HashMap
 import java.net.URI
-import scala.collection.JavaConversions._
 
 object CommandLine {
+  case class Params(
```

Comment:
That's smth to discuss; currently we have a bit different argument set, so that can be done probably in a separate PR if there would be such necessity?